src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,67 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(text) 
+{
+    //if the parameter type is not a valid string throw an error 
+    if(typeof text !== "string") 
+    {
+        throw `Please enter a valid string`
+    }
+    //variable to match any letter of alphabet
+    const regexToValidateText = /[a-z]/i; //flag 'i' to ignore case
+    
+    //loop through the input text to check valid input
+    for (let letter of text)
+    {
+        /*if you test any letter of the text and its not an alphabeth, throw 
+        *an error prompting the user to enter a valid input to the function */
+        if (!regexToValidateText.test(letter))
+        {
+            return `Please enter a valid string`;
+        }
+    }
```

Comment:
This doesn't actually throw an error. And there are prolly ways to validate the whole string is valid with regex without checking every character individually.